src/MultipartStreamBuilder.php

```diff
@@ -93,8 +93,8 @@ public function addData($resource, array $headers = [])
      *
      * @param string                                                      $name     the formpost name
      * @param string|resource|StreamInterface                             $resource
-     * @param array{'headers': array<string, string>, 'filename': string} $options
-     *
```

Comment:
please keep this blank line, otherwise the cs fixer makes some unfortunate requirement

Reply:
Oops fixed